vars/buildDockerImageForPerconaLab.groovy

```diff
@@ -0,0 +1,17 @@
+def call(version) {
+    withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]){
+    sh """
+        unset PSMDB_VERSION
+        unset MAJOR_VERSION
+        git clone https://github.com/percona/percona-docker.git
```

Comment:
can we make this as a param so we will be able to use private repos in case it is needed